config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1112,7 +1112,9 @@
 # Default: You have entered an email address ending with "{1}", which is restricted. Please try again with a different email address.
 # user.email.invalid.domain.message=You have entered a {0} email address ending with "{1}". Please log in with your existing {0} account.
 
-
+# Allow users with duplicated email
```

Comment:
Let's change this descriptive line to:
"Allow users to create new accounts with email addresses already being used (a duplicate email address)"